scout/adapter/mongo/variant_loader.py

```diff
@@ -543,6 +548,44 @@ def _load_variants(
 
         return nr_inserted
 
+    def _is_causative_other(
+        self,
+        variant: cyvcf2.Variant,
+        category: str = "snv",
+        build: str = "37",
+    ) -> bool:
+        """Check if variant is on the list of causatives from other cases.
+        All variants that have been marked causative will be loaded, even if the other case does not exist anymore, or has been reclassified.
+
+        Arguments:
+            variant(cyvcf2.Variant)
+            build(str): "37" or "38"
+
+        Returns:
+            is_managed(boolean)
+
+        """
+
+        coordinates = parse_coordinates(variant, category, build)
+
+        variant_prefix = parse_simple_id(
+            coordinates["chrom"],
+            str(coordinates["position"]),
+            coordinates["ref"],
+            coordinates["alt"],
+        )
+        clinical_variant = "".join([variant_prefix, "_clinical"])
+        research_variant = "".join([variant_prefix, "_research"])
+
+        var_causative_events_count = self.event_collection.find(
+            {
+                "verb": "mark_causative",
```

Comment:
Who cares about partial causatives?

Reply:
You apparently? 😸

Reply:
> You apparently? 😸

Naa I don't care!

Reply:
I added them anyway though the clearcut ones are likely the more important. Thank you!